tests/qml/helper.h

```diff
@@ -0,0 +1,44 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+#ifndef TESTHELPER_H
+#define TESTHELPER_H
+
+#include "../../src/closeeventhandler.h"
+#include "../../src/models/whatsnewmodel.h"
+
+#include <QObject>
+#include <QQmlEngine>
+#include <QtQuickTest>
+
+class TestHelper final : public QObject {
+  Q_OBJECT
+
+ public:
+  TestHelper();
+  static TestHelper* instance();
+  Q_INVOKABLE void triggerAboutToQuit();
+  Q_INVOKABLE void triggerInitializeGlean();
+  Q_INVOKABLE void triggerRecordGleanEvent(const QString& event);
+  Q_INVOKABLE void triggerSetGleanSourceTags(const QStringList& tags);
+  Q_INVOKABLE QString osVersion();
+  Q_INVOKABLE QString architecture();
+  Q_PROPERTY(bool mainWindowLoadedCalled READ mainWindowLoadedCalled)
+
+  CloseEventHandler closeEventHandler;
```

Comment:
Why this here and not as `whatsNewModel`? I'm OK with this, it's just style inconsistency :)

Reply:
I totally agree about the inconsistency. Will fix.